mods/voidmulti.sc2mod/base.sc2data/gamedata/effectdata.xml

```diff
@@ -1826,7 +1839,7 @@
     </CEffectModifyUnit>
     <CEffectModifyUnit id="ImmortalBarrierCDFix">
         <ValidatorArray value="BarrierOnCooldown"/>
-        <Cost Behavior="ImmortalBarrierSupressed" CooldownOperation="Add" CooldownTimeUse="8"/>
+        <Cost Behavior="ImmortalBarrierSupressed" CooldownOperation="Add" CooldownTimeUse="11"/>
```

Comment:
Raven's Interference Matrix duration reduced from 11 to 8 seconds

When Interference Matrix expires, it adds 11 seconds to Immortal's Barrier when the Barrier is on cooldown. Ok, I would have added it when Interference Matrix starts.
This looks like a buggy fix. What if the Barrier cooldown ends during Interference Matrix is active? I would expect this to be executed as initial effect and not when Interference Matrix stops. I suspect that this attempts to fix the cooldown ticking down during Interference Matrix when they want the cooldown to pause...

At least, these value changes here do not make sense!